### ISSUE TITLE
Account sub-headings not linking to parent heading

### ISSUE THREAD

**Issue by howardlowndes** [ledgersmb/LedgerSMB#7802]
### Version

Version 1.11.4

### What browsers are you seeing the problem on?

This problem isn't browser related

### What happened?

I am trying to set up a CoA in the subject version and I am setting up the headings before I set up the accounts in order to be sure of the structure that I want. I notice that the sub-heading levels are not linking to the relevant parent despite the previously created headings being displayed in the drop down and being selected. 

SQL follows:
```
PlayAMS=# select * from account_heading;
                         PlayAMS
 id | accno | parent_id |     description     | category 
----+-------+-----------+---------------------+----------
  1 | 1     |           | ASSETS              | 
  2 | 2     |           | LIABILITIES         | 
  3 | 3     |           | EQUITY              | 
  4 | 4     |           | INCOME              | 
  5 | 5     |           | EXPENSES            | 
  6 | 1.10  |           | HUME BANK           | 
  7 | 1.20  |           | NON-BANK            | 
  8 | 1.40  |           | PLANT & MACHINERY   | 
  9 | 1.50  |           | OFFICE EQUIPMENT    | 
 10 | 1.60  |           | INVENTORY           | 
 11 | 1.30  |           | ACCOUNTS RECEIVABLE | 
(11 rows)
```
Rows 6 thru 11 should all have the value "1" in the parent_id field.

![2023-12-10-211558_1360x768_scrot](https://github.com/ledgersmb/LedgerSMB/assets/52798206/c460e9b6-9af0-4cc3-a51d-1669b165c4db)

### What should have happened?

The parent selected from the presented drop down list should be written into the parent_id field.

**Comment by ehuelsmann** [comment]
Thanks for filing. I'll have a look at it right now.

**Comment by ehuelsmann** [comment]
This will be released as part of 1.11.6.
